modules/pruning-images.adoc

```diff
@@ -66,8 +67,13 @@ status:
 <9> `startingDeadlineSeconds`: Start deadline for `CronJob`. This is an optional field.
 <10> `successfulJobsHistoryLimit`: The maximum number of successful jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
 <11> `failedJobsHistoryLimit`: The maximum number of failed jobs to retain. Must be `>= 1` to ensure metrics are reported. Defaults to `3` if not set.
-<12> `observedGeneration`: The generation observed by the Operator.
-<13> `conditions`: The standard condition objects with the following types:
+<12> `logLevel`: Sets the level of log output for the pruner job. Defaults to  `Normal`.
```

Comment:
You could shorten this to the following:

 <12> `logLevel`: Sets the log level for the pruner job. Defaults to  `Normal`.